### PR TITLE
Changelog: 0.3.2

### PR DESCRIPTION
**Pull request by ax3l**
Adds the change log for release `0.3.2`. Please review it for typos and unclarities.

The title and abstract will as always go into the release title & test and on the user mailing list announcement.

Thanks to @ax3l, @psychocoderHPC, @sbastrakov and @Fettpet for contributions to this release!

(Please ignore the compile suite state, manual tests passed: #2525)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,47 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.3.2
+-----
+**Date:** 2018-02-16
+
+Phase Space Momentum, ADIOS 1 Particle Dumps & Field Names
```
Comment (PrometheusPi) [inline CHANGELOG.md]: *optional:*
`1-Particle Dumps`
or
`One-Particle Dumps`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+ - tools:
+   - tbg: wrong quoting of `'` #2419
+   - CMake: false-positive on in-source build check #2407
+   - pic-configure: cmakeFlags return code #2323
+
+**Misc:**
+ - Hypnos (HZDR): new modules #2521 #2524
+
+Thanks to Axel Huebl, René Widera, Sergei Bastrakov and Sebastian Hahn
+for contributions to this release!
```
Comment (PrometheusPi) [inline CHANGELOG.md]: *optional:*
for contributions to this ...

Comment (ax3l) [reply CHANGELOG.md]: as last release: we are using this string for all releases

Comment (ax3l) [reply CHANGELOG.md]: offline:

ah, you wanted to say "for contribut*ing* to this.."

**Pull request merged**